deploy/docker-compose/template/dfget.template.yaml

```diff
@@ -8,27 +8,32 @@ gcInterval: 1m0s
 # WorkHome is working directory.
 # In linux, default value is /usr/local/dragonfly.
 # In macos(just for testing), default value is /Users/$USER/.dragonfly.
-workHome: ''
+# With docker compose deployment, the path located to homedir since it's running as a none root user.
+workHome: '/home/dragonfly/.dragonfly'
```

Comment:
Please restore the default null value.

Reply:
it looks like the process would fail due to the permission issue of `/usr/local/dragonfly`

Reply:
You can document how to make changes, but do not affect the default configuration.